js/ui/list/ui.list.edit.strategy.grouped.js

```diff
@@ -19,7 +18,7 @@ const combineIndex = function(indices) {
 const splitIndex = function(combinedIndex) {
     return {
         group: combinedIndex >> SELECTION_SHIFT,
-        item: combinedIndex & SELECTION_MASK
+        item: combinedIndex & ((1 << SELECTION_SHIFT) - 1)
```

Comment:
It's good idea to specify a constant to not calculate this value each time

Reply:
Fair enough